joint-discovery-gatk4-fc.wdl

```diff
@@ -0,0 +1,896 @@
+  }
+}
+
+task GenotypeGVCFs {
+  File workspace_tar
+  String interval
+
+  String output_vcf_filename
+
+  File ref_fasta
+  File ref_fasta_index
+  File ref_dict
+
+  String dbsnp_vcf
+  
+  String docker_image
+  Int disk_size
+
+  command <<<
+    set -e
+
+    tar -xf ${workspace_tar}
+    WORKSPACE=$( basename ${workspace_tar} .tar)
+
+    /gatk/gatk-launch --javaOptions "-Xmx5g -Xms5g" \
+     GenotypeGVCFs \
+     -R ${ref_fasta} \
+     -O ${output_vcf_filename} \
+     -D ${dbsnp_vcf} \
+     -G StandardAnnotation \
+     --onlyOutputCallsStartingInIntervals \
+     -newQual \
+     -V gendb://$WORKSPACE \
+     -L ${interval}
+  >>>
+  runtime {
+    docker: docker_image
+    memory: "7 GB"
+    cpu: "2"
+    disks: "local-disk " + disk_size + " HDD"
+    preemptible: 5
+  }
+  output {
+    File output_vcf = "${output_vcf_filename}"
+    File output_vcf_index = "${output_vcf_filename}.tbi"
+  }
+}
+
+task HardFilterAndMakeSitesOnlyVcf {
+  File vcf
+  File vcf_index
+  Float excess_het_threshold
+
+  String variant_filtered_vcf_filename
+  String sites_only_vcf_filename
+
+  String docker_image
+  Int disk_size
+
+  command {
+    set -e
+
+    /gatk/gatk-launch --javaOptions "-Xmx3g -Xms3g" \
+      VariantFiltration \
+      --filterExpression "ExcessHet > ${excess_het_threshold}" \
+      --filterName ExcessHet \
+      -O ${variant_filtered_vcf_filename} \
+      -V ${vcf}
+
+    /gatk/gatk-launch --javaOptions "-Xmx3g -Xms3g"\
+      MakeSitesOnlyVcf \
+      --INPUT=${variant_filtered_vcf_filename} \
+      --OUTPUT=${sites_only_vcf_filename}
+
+  }
+  runtime {
+    docker: docker_image
+    memory: "3.5 GB"
+    cpu: "1"
+    disks: "local-disk " + disk_size + " HDD"
+    preemptible: 5
+  }
+  output {
+    File variant_filtered_vcf = "${variant_filtered_vcf_filename}"
+    File variant_filtered_vcf_index = "${variant_filtered_vcf_filename}.tbi"
+    File sites_only_vcf = "${sites_only_vcf_filename}"
+    File sites_only_vcf_index = "${sites_only_vcf_filename}.tbi"
+  }
+}
+
+task IndelsVariantRecalibrator {
+  String recalibration_filename
+  String tranches_filename
+
+  Array[String] recalibration_tranche_values
+  Array[String] recalibration_annotation_values
+
+  File sites_only_variant_filtered_vcf
+  File sites_only_variant_filtered_vcf_index
+
+  File mills_resource_vcf
+  File axiomPoly_resource_vcf
+  File dbsnp_resource_vcf
+  File mills_resource_vcf_index
+  File axiomPoly_resource_vcf_index
+  File dbsnp_resource_vcf_index
+
+  String docker_image
+  Int disk_size
+
+  command {
+    /gatk/gatk-launch --javaOptions "-Xmx24g -Xms24g" \
+      VariantRecalibrator \
+      -V ${sites_only_variant_filtered_vcf} \
+      -O ${recalibration_filename} \
+      --tranchesFile ${tranches_filename} \
+      -allPoly \
+      -tranche ${sep=' -tranche ' recalibration_tranche_values} \
+      -an ${sep=' -an ' recalibration_annotation_values} \
+      -mode INDEL \
+      --maxGaussians 6 \
```

Comment:
this seems to have been changed from 4 -> 6, any particular reason why? https://github.com/broadinstitute/dsde-pipelines/blob/develop/genomes_in_the_cloud/joint_genotyping_workflow/JointGenotypingWf.wdl#L426

Reply:
Left over edits from the using small input files. will fix this and double check the method in FC.